src/chesster.js

```diff
@@ -118,6 +108,137 @@ function leagueDMResponse(patterns, responseName) {
 leagueResponse(['captain guidelines'], 'formatCaptainGuidelinesResponse');
 leagueDMResponse(['captains', 'captain list'], 'formatCaptainsResponse');
 
+/* availability */
+/* [player <player-name> is] {available, unavailable} for round <round-number> in <league> */
+chesster.hears({
+    middleware: [ 
+        slack.withLeague
+    ],
+    patterns: [
+        'available',
+        'unavailable'
+    ],
+    messageTypes: [
+        'direct_message',
+        'direct_mention'
+    ]
+}, 
+function(bot, message){
+    var heltourOptions = message.league.options.heltour;
+    if(!heltourOptions){
+        winston.error("{} league doesn't have heltour options.".format(message.league.options.name));
+        return;
+    }
+
+    var commandDescription;
+    if(_.isEqual(_(message.text).split(' ').head(), "player")){
+        commandDescription = [
+            "player",
```

Comment:
The fact that we can add a playerName type to our system makes me like our system.